packages/circuits/circuits/buildOrderedTree.circom

```diff
@@ -59,9 +59,9 @@ template BuildOrderedTree(TREE_DEPTH, TREE_ARITY, FIELD_COUNT, OMT_R, EPK_R) {
    * TODO: find a way to not use a LessThan component
    * can use value differences/subtraction for this?
    **/
-  component lt_comp[TREE_ARITY**TREE_DEPTH];
+  component lt_comp[TREE_ARITY**TREE_DEPTH-1];
```

Comment:
I'm pretty sure that defining extra components without supplying inputs doesn't add constraints to the circuit. Regardless this refactor is probably fine.